engine/access/rest/routes/router.go

```diff
@@ -151,6 +151,11 @@ var Routes = []route{{
 	Pattern: "/accounts/{address}",
 	Name:    "getAccount",
 	Handler: GetAccount,
+}, {
+	Method:  http.MethodGet,
+	Pattern: "/accounts/{address}/balance",
```

Comment:
I think you'll need to update the `normalizeURL` method below to allow metics for this endpoint